lib/Elastica/Client.php

```diff
@@ -669,48 +684,23 @@ public function refreshAll()
     }
 
     /**
-     * logging.
-     *
-     * @param string|\Elastica\Request $context
-     *
-     * @throws Exception\RuntimeException
-     */
-    protected function _log($context)
```

Comment:
I'm worried here - this method could be overwritten by people extending `\Elastica\Client`. Its a BC break.

We should probably still do the logging in this method but deprecate it.